src/actions/index.js

```diff
@@ -0,0 +1,16 @@
+export function changeRestaurantField({ name = '', category = '', address = '' }) {
+  return {
+    type: 'changeRestaurantField',
+    payload: {
+      name,
+      category,
+      address,
+    },
+  };
+}
```

Comment:
```suggestion
export function changeRestaurantField({ name, value }) {
  return {
    type: 'changeRestaurantField',
    payload: {
      name,
      value,
    },
  };
}
```

지금과 같은 방식이면 파라미터가 끝없이 늘어날 겁니다.